src/tests/unit/endpoint_standard/test_endpoint_standard_enriched_events_facet.py

```diff
@@ -0,0 +1,201 @@
+"""Testing EnrichedEventFacet objects of cbc_sdk.endpoint_standard"""
+
+import pytest
+import logging
+from cbc_sdk.endpoint_standard import EnrichedEventFacet, EnrichedEventFacetQuery
+from cbc_sdk.rest_api import CBCloudAPI
+from cbc_sdk.errors import ObjectNotFoundError, ApiError
+from tests.unit.fixtures.CBCSDKMock import CBCSDKMock
+from tests.unit.fixtures.endpoint_standard.mock_enriched_events_facet import (POST_ENRICHED_EVENTS_FACET_SEARCH_JOB_RESP,
+                                                             GET_ENRICHED_EVENTS_FACET_SEARCH_JOB_RESULTS_RESP_1,
+                                                             GET_ENRICHED_EVENTS_FACET_SEARCH_JOB_RESULTS_RESP_2)
+
+
+log = logging.basicConfig(format='%(asctime)s %(levelname)s:%(message)s', level=logging.DEBUG, filename='log.txt')
+
+
+@pytest.fixture(scope="function")
+def cb():
+    """Create CBCloudAPI singleton"""
+    return CBCloudAPI(url="https://example.com",
+                      org_key="test",
+                      token="abcd/1234",
+                      ssl_verify=False)
+
+
+@pytest.fixture(scope="function")
+def cbcsdk_mock(monkeypatch, cb):
+    """Mocks CBC SDK for unit tests"""
+    return CBCSDKMock(monkeypatch, cb)
+
+
+# ==================================== UNIT TESTS BELOW ====================================
+
+def test_enriched_event_facet_select_where(cbcsdk_mock):
```

Comment:
There needs to be a test for using add_facet_field([str]) we should probably have a test for an unexpected value like a number or None as well

Reply:
Done.